Warn the player when taking off with outfits they can't carry (#7645)
endless-sky/endless-sky: tests/integration/config/plugins/integration-tests/data/tests/tests_common.txt

		call "Wait 55 steps"

test "Depart and Ignore Warning Message"
	status partial
	description "Helper function to depart from a planet when a message requiring acknowledgement via keyboard input is expected."
	sequence
		input
			key d
		# The game needs 2 gamesteps for processing the launch before commands like the jump command can be given.
		# Using empty inputs to trigger those game loops.
		input
		input

> 		# Dismiss the expected message

Reviewer: ```suggestion
		# Acknowledge the expected message
```
dismissing it would be pressing escape wouldnt it?
this does ok so it accepts whatever is asked

mb rename the test too

		input
			key "Return"

> 		# The ships zoom gets restored at a rate of .02 per step, so wait for a bit more than 50 steps to get to full zoom/full control.
> 		apply
> 			"test: Depart: zoom_restore_counter" = 55
> 		label zoom_restore_loop
> 		apply
> 			"test: Depart: zoom_restore_counter" -= 1
> 		input
> 		branch zoom_restore_loop
> 			"test: Depart: zoom_restore_counter" > 0

Reviewer: ```suggestion
		# The ships zoom gets restored at a rate of .02 per step, so wait for a bit more than 50 steps to get to full zoom/full control.
		call "Wait 55 steps"
```

		call "Wait 55 steps"

test "Depart and Abort on Warning Message"
	status partial
	description "Helper function to attempt departure from a planet, then stop when an expected warning message is given."
	sequence
		input
			key d
		# The game needs 2 gamesteps for processing the launch before commands like the jump command can be given.
		# Using empty inputs to trigger those game loops.
		input
		input

> 		# Acknowledge the expected message

Reviewer: ```suggestion
		# Ignore the expected message
```
this time you need it this way ^^
cancel works too, idk

Author: The first one is actually ignoring the warning, because it continues to take off. This accepts the warning and doesn't take off.
~~Time to add a way to customise the text in those buttons.~~
